### DISCUSSION TITLE
Updating xstate/inspect to be able to be run in codepen.io or other skypack environments

### DISCUSSION THREAD

**Discussion by SimeonC** [statelyai/xstate#3046] [Ideas] [answered]
I'd like to embed the inspector inside a codepen, but due to these lines of code skypack fails to import it. (This would also affect other ES6 browser only usages too I think)

https://github.com/statelyai/xstate/blob/673cdf693068a9a81558b9a59b32c937ddab867b/packages/xstate-inspect/src/browser.ts#L10-L15

In skypack this compiles to the following.
```js
import {createMachine, State, assign, interpret} from "/-/xstate@v4.29.0-j6UUodRhQ9SIrYL5N6zm/dist=es2019,mode=imports/optimized/xstate.js";
import {toSCXMLEvent, toEventObject, toObserver, interopSymbols} from "/-/xstate@v4.29.0-j6UUodRhQ9SIrYL5N6zm/dist=es2019,mode=imports/unoptimized/lib/utils.js";
```

I think the three options for fixing this are;

1. Change rollup to inline all xstate code into the inspect output. - not great for bundle size, but it is a dev util.
2. Change Xstate main package to export all these utils etc from the root, something like `import { utils: { toSCXMLEvent } } from 'xstate'`
3. Manually copy over the util functions to the inspect package so it doesn't rely on them.

**Comment by Andarist** [comment, marked as answer]
I'll look into fixing this today.

**Reply by SimeonC** [reply]
Awesome! I was gonna say I could try fix it myself with guidance, but if you guys want to handle that it's fine too 👍

**Reply by Andarist** [reply]
Oh, sorry - I've totally missed your reply here. I've created a PR to fix this here: https://github.com/statelyai/xstate/pull/3089
